### DISCUSSION TITLE
/!\ dbeaver data filtering problem /!\

### DISCUSSION THREAD

**Discussion by Julien699** [dbeaver/dbeaver#14834] [Q&A]
Hello I am a French user I am not used to github but I have to come here. I have encountered a very annoying problem since exactly version 21.1.4 I can no longer filter elements in a column of a table in my database. The problem is that it takes a very long time before the filter window is displayed whereas in versions prior to 21.1.4 it is almost instantaneous.

CPU: Ryzen 7 1800x

Ram: 16gb 3200mhz

Operating System: Windows 10 Pro 21H2 19044.1387

Dbeaver is installed on an ssd. I am connected to a mariadb database. The problem occurs with tables of +10,000 data. Engine used by the different tables: MyISAM / InnoDB

I don't know much about it so I give the most basic information. I just know that this problem exists since version 21.1.4 and because of it I cannot update dbeaver. if I press the filter logo dbeaver will stop responding then after 1-2 minutes it will show me the filter window it's very disturbing.

**Comment by LonwoLonwo** [comment]
We fixed this issue in DBeaver 21.3.2. 
Version will be available [here](https://dbeaver.io/) on 2021-12-27
